helm-charts/explorer/templates/explorer-configmap.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-config
+  labels:
```

Comment:
Namespace property missing. check other files as well.

Reply:
Added Namespace to all files